tests/textcase/plugin/plugin_spec.lua

```diff
@@ -113,6 +113,47 @@ describe("plugin", function()
     end
   end)
 
+  describe("visuel modes", function()
+    local test_cases = {
+      {
+        keys = "Vjjgan",
+        buffer_lines = {
+          "LoremIpsum DolorSit amet",
+          "lorem_ipsum dolor_sit amet",
+          "lorem-ipsum dolor-sit amet",
+        },
+        expected = {
+          "LOREM_IPSUM_DOLOR_SIT_AMET",
+          "LOREM_IPSUM_DOLOR_SIT_AMET",
+          "LOREM_IPSUM_DOLOR_SIT_AMET",
+        },
+      },
+      {
+        keys = "<C-V>eejgan",
+        buffer_lines = {
+          "LoremIpsum DolorSit amet",
+          "lorem_ipsum dolor_sit amet",
+          "lorem-ipsum dolor-sit amet",
+        },
+        expected = {
+          "LOREM_IPSUM_DOLOR_SIT amet",
+          "LOREM_IPSUM_DOLOR_Sit amet",
+          "lorem-ipsum dolor-sit amet",
+        },
+      },
+    }
+
+    for _, test_case in ipairs(test_cases) do
```

Comment:
This loop seems super effective for batch testing

Reply:
When creating a test helper, we should think about creating a method for it.